test/test_sparse.py

```diff
@@ -3632,9 +3632,9 @@ def test_sparse_sparse_mul(self, device, dtype, coalesced):
         nnz = 10
 
         def check(self, x, y):
-            res_sparse = x * y
             res_dense = x.to_dense() * y.to_dense()
-            self.assertEqual(res_sparse.to_dense(), res_dense)
+            self.assertEqual(res_dense, x * y)
+            self.assertEqual(res_dense, y * x)
```

Comment:
Was this failing prior to this PR?

Reply:
No, just a missing test for completeness.